.github/workflows/review.yml

```diff
@@ -20,6 +20,7 @@ env:
   K8S_READINESS_PATH: /api/readiness
   NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT: https://liikunta.hkih.production.geniem.io/graphql
   NEXT_PUBLIC_UNIFIED_SEARCH_GRAPHQL_ENDPOINT: https://unified-search.test.kuva.hel.ninja/search
+  NEXT_PUBLIC_NEXT_API_GRAPHQL_ENDPOINT: https://${{ env.K8S_NAMESPACE }}.${{ env.BASE_DOMAIN }}/api/graphql
```

Comment:
Review action crashes at the moment with error `The workflow is not valid. .github/workflows/review.yml (Line: 23, Col: 42): Unrecognized named-value: 'env'. Located at position 1 within expression: env.K8S_NAMESPACE`

Seems that this part of the file doesn't have access to `env` and throws an error. Maybe just move the value directly on the lines 37 & 54.

Reply:
I think I am referencing `env` while defining it, so it makes sense that this approach fails. I couldn't find any DRY approach to this either so inlining sounds like a good option for now.